File: [Compi1]HT2/Clase1.java
import java.util.Scanner;


public class Clase1{

	static Scanner sc = new Scanner(System.in);
	private int variable1 = 10/5+3;
	public void metodo1 (){
		System.out.println("el factorial de 5" + "es: "+factorial(variable1));
	int variable2 = sc.nextInt();
	System.out.println("------------------------------");
	System.out.println("El factorial de " + variable2 + " es: "+factorial(variable2));
	}
	public int factorial (int num){
	if(num==0){
		return 1

	}
	Else{
		return num*factorial(num-1)

	}

	}

}

File: [Compi1]HT2/Clase1.vb
Module Clase1
	Dim variable1 As Integer=10/5+3
	Public Sub metodo1()
		Console.WriteLine("el factorial de 5" & "es: " + factorial(variable1))
		variable2 As Integer = Console.ReadLine
		Console.WriteLine("------------------------------")
		Console.WriteLine("El factorial de "&variable2&" es: "+ factorial(variable2))
	End Sub
	Public Function factorial(ByVal num As Integer) As Integer
		If num = 0 Then
			Return 1
		Else
			Return num*factorial(num-1)
		End If
	End Function

End Module
